doc/guides/device_mgmt/ota.rst

```diff
@@ -0,0 +1,65 @@
+.. _ota:
+
+Over-the-Air Update
+###################
+
+Overview
+********
+
+Over-the-Air (OTA) Update is a method for delivering firmware updates to remote
+devices using a network connection. Although the name implies a wireless
+connection, devices receiving updates over a wired connection (such as Ethernet)
+are still commonly referred to by OTA. This approach requires server
+infrastructure to host the firmware binary and implement a method of signaling
+when an update is available.
+
+The :ref:`dfu` section discusses upgrading Zephyr firmware using MCUboot. The
+same method can be used as part of OTA. The binary is first downloaded
+into an unoccupied code partition, then flashed using the `MCUboot bootloader`_
+process.
+
+Examples of OTA
+***************
+
+Golioth
+=======
+
+`Golioth`_ is an IoT management platform that includes Zephyr OTA updates.
+Devices are configured to observe your available firmware revisions on the
+Golioth Cloud. When a new version is available, the device downloads and flashes
+the binary.
+
+Security is a concern with OTA updates. In this implementation, the connection
+between cloud and device is secured using TLS/DTLS, and the signed firmware binary
+is confirmed by MCUboot before flashing occurs.
+
+1. A working sample can be found on the `Golioth Zephyr-SDK repository`_
+2. The `Golioth OTA documentation`_ includes complete information about the
+   versioning process
+
+Eclipse hawkBit |trade|
+=======================
+
+`Eclipse hawkBit`_ |trade| is a domain independent back-end framework for rolling out
+software updates to constrained edge devices as well as more powerful
+controllers and gateways connected to IP based networking infrastructure.
+
+There is a :ref:`hawkbit-api-sample` included in the
+Zephyr :ref:`mgmt-samples` section.
+
+UpdateHub
+=========
+
+`UpdateHub`_ is an enterprise-grade solution which makes it simple to remotely
+update all your Linux-based devices in the field, with maximum security and
+efficiency, while you focus on adding value to your product.
```

Comment:
IMO this is **way** too fluffy for a technical page; was it copy/pasted from some marketing copy for UpdateHub? I think that's not appropriate if so. Let's focus on the technical aspects of the solution here, please.

Also, why do we care about updating Linux-based devices here? This is Zephyr, not Linux.

Reply:
Good criticism, and yes I just pulled the paragraph from their web page since I'm not familiar with the project. I have reworked this entry following your feedback.